Wordle.Api/Wordle.Api/Controllers/WordController.cs

```diff
@@ -30,13 +32,13 @@ public async Task<IEnumerable<Word>> GetManyWords(int? count)
         }
 
         [HttpPost]
-        public async Task<Word> AddWord(string newWord, bool isCommon)
+         public async Task<Response<Word>> AddWord(string newWord, bool isCommon)
```

Comment:
You should either add an auth policy to this or remove it in favor of your other add word function